torch/_inductor/lowering.py

```diff
@@ -1155,6 +1156,35 @@ def inner_fn(idx):
     )
 
 
+@register_lowering(
+    quantized_wrapper.wrapped_fbgemm_pack_gemm_matrix_fp16.default,
+    type_promotion_kind=None,
+)
+def wrapped_linear_pack_weight_fp16(weight):
+    curr_op = quantized_wrapper.wrapped_fbgemm_pack_gemm_matrix_fp16.default
+
+    add_needs_realized_inputs(curr_op)
+    fallbacks.add(curr_op)
+
+    return pytree.tree_map(TensorBox.create, ir.FallbackKernel.create(curr_op, weight))
+
+
+@register_lowering(
+    quantized_wrapper.wrapped_fbgemm_linear_fp16_weight.default,
+    type_promotion_kind=None,
+)
+def linear_fp16_weight(input, weight, bias, out_channel):
+    curr_op = quantized_wrapper.wrapped_fbgemm_linear_fp16_weight.default
+
+    add_needs_realized_inputs(curr_op)
+    fallbacks.add(curr_op)
+
+    return pytree.tree_map(
+        TensorBox.create,
+        ir.FallbackKernel.create(curr_op, input, weight, bias, out_channel),
+    )
```

Comment:
Can we make them implicit fallback so that you don't have to register lowering for them explicitly here?

Reply:
Hi @jgong5, I addressed your comment in the new PR: #122939 
(Sorry for making a new PR as I need to deal with some OSS/internal syncing issue.)

Thanks!